src/main/java/com/bookstore/bookstoreapi/data/entity/Author.java

```diff
@@ -35,4 +35,8 @@ public class Author {
 
   @Column(name = "Publisher", nullable = false)
   private String publisher;
+
+  public void setFirstName(String firstName) {
```

Comment:
why do you have setters? we are using `Lombok`'s `@Data` annotation to do that job for us and remove the boilerplate code.